docs/reference/operations/create_accounts.md

```diff
@@ -0,0 +1,112 @@
+[`Account.flags.reserved`](../accounts.md#flagsreserved) is nonzero, but must be zero.
+
+### `reserved_field`
+The account was not created.
+[`Account.reserved`](../accounts.md#reserved) is nonzero, but must be zero.
+
+### `id_must_not_be_zero`
+The account was not created.
+[`Account.id`](../accounts.md#id) is zero, which is a reserved value.
+
+### `id_must_not_be_int_max`
+The account was not created.
+[`Account.id`](../accounts.md#id) is `2^128 - 1`, which is a reserved value.
+
+### `ledger_must_not_be_zero`
+The account was not created.
+[`Account.ledger`](../accounts.md#ledger) is zero, but must be nonzero.
+
+### `code_must_not_be_zero`
+The account was not created.
+[`Account.code`](../accounts.md#code) is zero, but must be nonzero.
+
+### `debits_pending_must_be_zero`
+The account was not created.
+[`Account.debits_pending`](../accounts.md#debits_pending) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `debits_posted_must_be_zero`
+The account was not created.
+[`Account.debits_posted`](../accounts.md#debits_posted) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `credits_pending_must_be_zero`
+The account was not created.
+[`Account.credits_pending`](../accounts.md#credits_pending) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `credits_posted_must_be_zero`
+The account was not created.
+[`Account.credits_posted`](../accounts.md#credits_posted) is nonzero, but must be zero.
+
+An account's debits and credits are only modified by transfers.
+
+### `mutually_exclusive_flags`
+The account was not created.
+An account cannot be created with the specified combination of
+[`Account.flags`](../accounts.md#flags).
+
+The following flags are mutually exclusive:
+- [`Account.flags.debits_must_not_exceed_credits`](../accounts.md#flagsdebits_must_not_exceed_credits)
+- [`Account.flags.credits_must_not_exceed_debits`](../accounts.md#flagscredits_must_not_exceed_debits)
+
+### `exists_with_different_flags`
+The account already exists, but with different [`flags`](../accounts.md#flags).
+
+### `exists_with_different_user_data`
+The account already exists, but with different [`user_data`](../accounts.md#user_data).
+
+### `exists_with_different_ledger`
+The account already exists, but with different [`ledger`](../accounts.md#ledger).
+
+### `exists_with_different_code`
+The account already exists, but with different [`code`](../accounts.md#code).
+
+### `exists`
+The account already exists.
+
+With the possible exception of the following fields, the existing account is identical to the
+account in the request:
+- `debits_pending`
+- `debits_posted`
+- `credits_pending`
+- `credits_posted`
+
+To correctly recover from application crashes
+[many applications](../../usage/integration.md#conistency-with-foreign-databases) should handle
+`exists` exactly as [`ok`](#ok).
```

Comment:
Dumb question: why does `exists` exist then if the correct way to handle it is only ever to treat it as ok?

Reply:
Not a dumb question at all — actually an extremely tricky one!

- Treating `exists` as `ok` is correct in this particular context. (i.e. when coordinating with a foreign database as the "conistency-with-foreign-databases" describes).
- But there are other situations where `exists` should be treated differently. I just haven't documented them yet (they are quite niche — this section describes how integration will work most of the time).
- Part of it is also TB's philosophy of returning the most detailed result code it can. (e.g. rather than just `error`, we might send `reserved_flag`).